guides/common/modules/proc_using-autocompletion-in-templates.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Autocompletion_in_Templates_{context}"]
+= Using Autocompletion in Templates
+
+Using the autocompletion option, you can access a list of all available macros and information about using it, in the template editor itself.
+This works for all {ProjectName} templates.
+
+.Procedure
+. Navigate to either *Hosts* > *Partition tables*, *Hosts* > *Provisioning templates*, or *Hosts* > *Job templates*.
+. Click the *settings* icon at the top right corner of the template editor and select *Autocompletion*.
+. Press `*Ctrl*` + `*Space*` in the template editor to access a list of all available macros.
+. You can narrow down the list of macros by typing in more information about what you are looking for.
+. For example, if you are looking for a method to list the ID of the content source for a host, you can type `host` and check the list of available macros for content source.
```

Comment:
```suggestion
You can narrow down the list of macros by typing in more information about what you are looking for.
For example, if you are looking for a method to list the ID of the content source for a host, you can type `host` and check the list of available macros for content source.
```

I don't think these are steps to follow. IMHO there should only be a numbered list item if the user has to do something.